openmc/mgxs_library.py

```diff
@@ -0,0 +1,705 @@
+from collections import Iterable
+from numbers import Real, Integral
+from xml.etree import ElementTree as ET
+import warnings
+import sys
+if sys.version_info[0] >= 3:
+    basestring = str
+
+import numpy as np
+
+import openmc
+from openmc.mgxs import EnergyGroups
+from openmc.checkvalue import check_type, check_value, check_greater_than, \
+                              check_iterable_type
+from openmc.clean_xml import *
+
+# MGXS Representations supported by OpenMC
```

Comment:
Perhaps clarify this comment that `REPRESENTATIONS` is specific to the angular discretization used for the MGXS.